app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupsFragment.java

```diff
@@ -4,20 +4,39 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.TextView;
 
 import androidx.fragment.app.Fragment;
 
 import com.randomappsinc.studentpicker.R;
 
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.Unbinder;
+
 public class GroupsFragment extends Fragment {
 
     public static GroupsFragment getInstance() {
         return new GroupsFragment();
     }
 
+    @BindView(R.id.no_groups) TextView noGroups;
+
+    private Unbinder unbinder;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.fragment_groups, container, false);
+        unbinder = ButterKnife.bind(this,rootView);
+
+        noGroups.setVisibility(View.VISIBLE);
```

Comment:
We should do this in XML. We should separate layout logic into the XML and out of the Java as much as possible to focus their responsibilities.

Reply:
it's just a temporary line, I will remove it after complete the scenario anyways, Should I put it in the XML now.

Reply:
Ah, you did it for the demo. Sounds good, we can keep it for now. :)